src/main/resources/db/migration/V1_0__create_schema.sql

```diff
@@ -0,0 +1,22 @@
+create table roles
+(
+  id serial not null,
+  role varchar(20) not null default '',
+  primary key (id)
+);
+
+create table users
+(
+  id serial not null,
+  email varchar(50) not null unique,
+  password varchar(150),
+  primary key (id)
+);
+
+create table user_role
+(
+  user_id int not null,
```

Comment:
I don’t remember if the table fields should match the parent

Reply:
they should